README.md

```diff
@@ -3,4 +3,13 @@
 * `base` contains the base image with some scripts, qemu and dnsmasq
 * `centos7` adds a vagrant centos7 box to the image
 * `cli` contains a tool for provisioning, running and managing the containerized clusters
-* `kubeadm-1.9.3` k8s-1.9.3 cluster based on the centos7 image, provisioned with kubeadm
+* `k8s-1.9.3` k8s-1.9.3 cluster based on the centos7 image, provisioned with kubeadm
+* `os-3.9` os-3.9 cluster based on the centos7 image, provisioned with openshift-ansible
+
+## Versions to use
+
+* `kubevirtci/cli`: `sha256:e640c6a23e75eeb98ac8230c0542c83a0a4a556bed72d99fa89b3fb68c3976d5`
```

Comment:
@cynepco3hahue that is already the one with nfs.

Reply:
Will update my PR